entrypoint/clusterreset/clusterreset_test.go

```diff
@@ -0,0 +1,93 @@
+package clusterreset
+
+import (
+	"fmt"
+
+	"github.com/rancher/distros-test-framework/pkg/assert"
+	"github.com/rancher/distros-test-framework/pkg/testcase"
+
+	. "github.com/onsi/ginkgo/v2"
+)
+
+var _ = Describe("Test:", func() {
+
+	It("Start Up with no issues", func() {
+		testcase.TestBuildCluster(GinkgoT())
+	})
+
+	It("Validate Nodes", func() {
+		testcase.TestNodeStatus(
+			assert.NodeAssertReadyStatus(),
+			nil,
+		)
+	})
+
+	It("Validate Pods", func() {
+		testcase.TestPodStatus(
+			assert.PodAssertRestart(),
+			assert.PodAssertReady(),
+			assert.PodAssertStatus(),
+		)
+	})
+
+	It("Verifies ClusterIP Service", func() {
+		testcase.TestServiceClusterIp(true, true)
+	})
+
+	It("Verifies NodePort Service Before Reset", func() {
```

Comment:
be consistent and clear on naming.

if you have same tests running pre and pos some other test, in your case cluster reset, name it accordingly 
pre ...pos ....etc.